functions-and-operators/information-functions.md

```diff
@@ -25,6 +26,224 @@ TiDB supports most of the [information functions](https://dev.mysql.com/doc/refm
 | [`USER()`](https://dev.mysql.com/doc/refman/8.0/en/information-functions.html#function_user) | Return the user name and host name provided by the client    |
 | [`VERSION()`](https://dev.mysql.com/doc/refman/8.0/en/information-functions.html#function_version) | Return a string that indicates the MySQL server version   |
 
+### BENCHMARK()
+
+The `BENCHMARK()` function runs the expression a number of times.
+
+```sql
+SELECT BENCHMARK(5, SLEEP(2));
+```
+
+```
++------------------------+
+| BENCHMARK(5, SLEEP(2)) |
++------------------------+
+|                      0 |
++------------------------+
+1 row in set (10.00 sec)
+```
+
+### CONNECTION_ID()
+
+This returns the connection id of the session.
+
+```sql
+SELECT CONNECTION_ID();
+```
+
+```
++-----------------+
+| CONNECTION_ID() |
++-----------------+
+|       322961414 |
++-----------------+
+1 row in set (0.00 sec)
+```
+
+### CURRENT_ROLE()
+
+This is to get the current [role](/role-based-access-control.md).
+
+```sql
+SELECT CURRENT_ROLE();
+```
+
+```
++----------------+
+| CURRENT_ROLE() |
++----------------+
+| NONE           |
++----------------+
+1 row in set (0.00 sec)
+```
+
+### CURRENT_USER()
+
+This returns the account that is using the session.
+
+```sql
+SELECT CURRENT_USER();
+```
+
+```
++----------------+
+| CURRENT_USER() |
++----------------+
+| root@%         |
++----------------+
+1 row in set (0.00 sec)
+```
+
+### DATABASE()
+
+This returns the database schema that the current session is using.
+
+```sql
+SELECT DATABASE();
+```
+
+```
++------------+
+| DATABASE() |
++------------+
+| test       |
++------------+
+1 row in set (0.00 sec)
+```
+
+### FOUND_ROWS()
+
+```sql
+SELECT 1 UNION ALL SELECT 2;
+```
+
+```
++------+
+| 1    |
++------+
+|    2 |
+|    1 |
++------+
+2 rows in set (0.01 sec)
+```
+
+```sql
+SELECT FOUND_ROWS();
+```
+
+```
++--------------+
+| FOUND_ROWS() |
++--------------+
+|            2 |
++--------------+
+1 row in set (0.00 sec)
+```
+
+> **Note**
+>
+> `SQL_CALC_FOUND_ROWS` is only accepted if [`tidb_enable_noop_functions`](/system-variables.md#tidb_enable_noop_functions-new-in-v40) is set. This query modifier has been deprecated in MySQL 8.0.17.
+
+### LAST_INSERT_ID()
+
+This returns the ID of the last row that was inserted. This works with [`AUTO_INCREMENT`](/auto-increment.md) and [`AUTO_RANDOM`](/auto-random.md) columns.
+
+```
+mysql> CREATE TABLE t1(id BIGINT UNSIGNED PRIMARY KEY AUTO_RANDOM);
```

Comment:
Even though this is more interesting, I think an example with AUTO_INCREMENT is more straight forward.